esp/platform/esptrace.h

```diff
@@ -0,0 +1,73 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2024 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#pragma once
+
+#include "jtrace.hpp"
+#include <initializer_list>
+
+// Refer to helm/examples/esp/README.md for more information on the trace options.
+
+constexpr const char* propTraceFlags = "traceFlags";
+constexpr const char* propTraceLevel = "traceLevel";
+
+// ESP process names for use with options mapping
+constexpr const char* eclwatchApplication = "eclwatch";
+constexpr const char* eclservicesApplication = "eclservices";
+constexpr const char* eclqueriesApplication = "eclqueires";
```

Comment:
Unused, and has a typo in the string.

Reply:
I suppose the definitions are premature. I defined them so mapTraceOptions will eventually be able to use different option lists for distinct ESPs. Not sure how many new options will be defined for most platform ESPs, but I have a lot in mind for esdl and esdl-sandbox (when/if we return focus to DESDL) that might also extend into loggingservice.

Are you asking for removal until needed, or is fixing the type sufficient?

Reply:
I don't mind, probably cleaner to remove.
The danger with adding constants or other items that are not actually used, is that when they come to be used they are not checked, because they are assumed to be correct because they are already checked in, but equally there is a danger that items that are not used aren't checked as carefully when they are merged.